Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -1183,6 +1183,13 @@ working, take a look at `this KEP
     free mode, make sure that default Kubernetes services like ``kube-dns`` and ``kubernetes``
     have the required label value.
 
+Outside access to ClusterIP services
```

Comment:
nit: External access?

Reply:
Reworded it a bit and included link to k8s documentation on ClusterIP services.